docs/ftldns/configfile.md

```diff
@@ -377,4 +383,19 @@ Should FTL translate its own stack addresses into code lines during the bug back
 
 Temporary flag that may print additional information. This debug flag is meant to be used whenever needed for temporary investigations. The logged content may change without further notice at any time.
 
+### Deprecated options
+
+#### `REPLY_ADDR4=` (unset by default, PR [#965](https://github.com/pi-hole/FTL/pull/965)) {#reply_addr4 data-toc-label='Force A reply'}
+
+*This option is deprecated and may be removed in future versions, please use `BLOCK_IPV4` and `LOCAL_IPV4` instead*
+
+If neither `BLOCK_IPV4` nor `LOCAL_IPV4` are set, this setting use used to set both of them. If any of the two is set, this setting is ignored altogether.
```

Comment:
```suggestion
If neither `BLOCK_IPV4` nor `LOCAL_IPV4` are set, this setting used to set both of them. If any of the two is set, this setting is ignored altogether.
```

Reply:
I believe "...this setting _is_ used..." was intended... For the second sentence, suggest "If either are set..." or "If either of the two _are_ set..."